### PR TITLE
chore: update apisix-ingress-controller to 1.7.0

### PR DESCRIPTION
**Pull request by lingsamuel**
ApisixRoute v2beta1 and v2beta2 are removed

### CODE REVIEW

File: charts/apisix-ingress-controller/Chart.yaml [hunk 1]
```diff
@@ -24,8 +24,8 @@ keywords:
   - nginx
   - crd
 type: application
-version: 0.11.7
-appVersion: 1.6.1
+version: 0.11.8
```
Comment (tao12345666333) [inline charts/apisix-ingress-controller/Chart.yaml:27]:
```suggestion
version: 0.12.0
```

**Review by soulbird — APPROVED** [review]

**Review by AlinsRan — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]

**Comment by tao12345666333** [comment]
Please wait

**Comment by tao12345666333** [comment]
CRD need to be updated

**Comment by lingsamuel** [comment]
CRDs are split and re-synced from APISIX Ingress Controller repo.
FYI: they are formatted by [mikefarah/yq](https://github.com/mikefarah/yq) `yq -i 'sort_keys(..)'` command to do diff things to ensure compatibility.

**Pull request merged**